src/mount/readdata.cc

```diff
@@ -383,9 +407,12 @@ void* read_data_delayed_ops(void *arg) {
 
 	pthread_setname_np(pthread_self(), "readDelayedOps");
 
+	std::unique_lock gMutexLock(gMutex, std::defer_lock),
+	    gUsedReadCacheMemoryLock(gReadCacheMemoryMutex, std::defer_lock);
```

Comment:
Please use two lines for two declarations.

Reply:
Done